example/zping-client/zping-client.go

```diff
@@ -135,8 +136,8 @@ func main() {
 		//fmt.Println(file)
 		configFile, err := config.NewFromFile(file)
 		if err != nil {
```

Comment:
We usually use a pattern like: `logrus.WithError(err).Error("Problem loading config file")`

Reply:
This is pattern used in chat-server
if err != nil {
		logrus.Errorf("Error binding service %+v", err)
		panic(err)
	}

Reply:
Will change to pattern as suggested